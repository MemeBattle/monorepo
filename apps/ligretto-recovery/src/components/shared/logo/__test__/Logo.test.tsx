import React from 'react'
import renderer from 'react-test-renderer'

import { Logo } from '../index'

describe('Logo', () => {
  it('should render correctly', () => {
    const component = renderer.create(<Logo />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
