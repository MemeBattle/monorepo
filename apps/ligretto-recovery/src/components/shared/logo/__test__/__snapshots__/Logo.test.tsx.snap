// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render correctly 1`] = `
<span
  className="logo"
>
  Ligretto Game
</span>
`;
